xl/main.py

```diff
@@ -644,6 +644,11 @@ def __init(self):
 
         engine.migrate()
 
+        # Migrate flac tempo tag
```

Comment:
Is this just for FLAC or also Ogg (/Vorbis) files?

Reply:
It's for both. 
But the migration only sets the setting to true in case of a first run and false otherwise.